docs/reference/eql/eql-search-api.asciidoc

```diff
@@ -433,10 +433,25 @@ Name of the index containing the event.
 
 `_id`::
 (string)
-(string)
 Unique identifier for the event.
 This ID is only unique within the index.
 
+`_version`::
+(integer)
+Version of the document (event). This version is incremented each time the document is
+updated.
+
+`_seq_no`::
+(integer)
+Sequence number assigned to the document (event).
++
+Sequence numbers are used to ensure an older version of a document
+doesn’t overwrite a newer version. See <<optimistic-concurrency-control>>.
+
+`_primary_term`::
+(integer)
+Primary term assigned to the document. See <<optimistic-concurrency-control>>.
+
```

Comment:
This is nested under the `sequences` property, which is documented as only returned for sequence queries.